.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
Actually it seems we're totally off (https://www.relishapp.com/rspec/rspec-core/docs/configuration/read-command-line-configuration-options-from-files)

> RSpec reads command line configuration options from files in three different
locations:
> - Local: ./.rspec-local (i.e. in the project's root directory, can be
gitignored)
> - Project: ./.rspec (i.e. in the project's root directory, usually
checked into the project)
> - Global: ~/.rspec (i.e. in the user's home directory)